tests/lib/c_lib/src/main.c

```diff
@@ -170,22 +187,31 @@ void test_strlen(void)
 {
 	(void)memset(buffer, '\0', BUFSIZE);
 	(void)memset(buffer, 'b', 5); /* 5 is BUFSIZE / 2 */
-	zassert_equal(strlen(buffer), 5, "strlen");
+	zassert_equal(strlen(buffer), 5, "strlen failed");
+	zassert_equal(strnlen(buffer, 6), 5, "strnlen failed");
+	zassert_equal(strnlen(buffer, 4), 4, "strnlen failed");
 }
 
 /**
  *
  * @brief Test string compare function
  *
+ * @see strcmp(), strncasecmp().
+ *
  */
```

Comment:
Could you add a `@ingroup` tag for all testcases' descriptions? It is useful for testcases.